### PR TITLE
update mapping analysis entity and property information

### PR DESCRIPTION
**Pull request by YannanGao-gs**
#### What does this PR do / why is it needed ?

update mapping analysis entity and property information

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: legend-engine-xts-analytics/legend-engine-xts-analytics-mapping/legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverage/analyticsHelper.pure [hunk 1]
```diff
@@ -43,7 +51,14 @@ function meta::analytics::mapping::modelCoverage::getRealProperty(property:Abstr
 
 function <<access.private>> meta::analytics::mapping::modelCoverage::buildPropertyInfo(p:AbstractProperty<Any>[1]):PropertyInfo[1]
 {
-   ^PropertyInfo(name = $p->originalPropertyNameForEdgePointProperty()->toOne(), multiplicity = $p.multiplicity);
+   let name = $p->originalPropertyNameForEdgePointProperty()->toOne();
+   ^PropertyInfo(
+      taggedValues = $p.taggedValues->map(val | ^TaggedValueInfo(tagOwner = $val.tag.profile->elementToPath(), tagValue = $val.tag.value, value = $val.value)),
```
Comment (gs-jp1) [inline legend-engine-xts-analytics/legend-engine-xts-analytics-mapping/legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverage/analyticsHelper.pure:56]: would be nice to make this a function that takes in taggedValues so can be reused below. same for stereotypes

File: legend-engine-xts-analytics/legend-engine-xts-analytics-mapping/legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverage/analytics.pure [hunk 2]
```diff
@@ -47,24 +47,56 @@ Enum meta::analytics::mapping::modelCoverage::MappedPropertyType
...
 Class meta::analytics::mapping::modelCoverage::MappedPropertyInfo
 {
   type : MappedPropertyType[1];
-  multiplicity: Multiplicity[1];
+  multiplicity: Multiplicity[1]; 
+  qualifiers : MappedPropertyQualifier[*];
+  taggedValues: TaggedValueInfo[*];
+  stereotypes : StereotypeInfo[*];
+  classPath: String[*]; // use * to make this optional 
+  isQualifiedProp: Boolean[*]; // use * to make this optional, when it's false or undefined, it means it's not a qualified prop
```
Comment (gs-jp1) [inline legend-engine-xts-analytics/legend-engine-xts-analytics-mapping/legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverage/analytics.pure:99]: should be 0..1?

File: legend-engine-xts-analytics/legend-engine-xts-analytics-mapping/legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverage/mappedEntityBuilder.pure [hunk 3]
```diff
@@ -74,15 +74,33 @@ function <<access.private>> meta::analytics::mapping::modelCoverage::buildProper
...
+          taggedValues = $propertyInfo.taggedValues,
+          qualifiers = $propertyInfo.qualifiers,
+          isQualifiedProp = $isQualifiedProp,
+          stereotypes = $propertyInfo.stereotypes),
+          | [])),
       et:Enumeration<Any>[1] |
          ^EnumMappedProperty(
-            name = $propertyInfo.name, enumPath = $type->elementToPath(), info = if($config.returnMappedPropertyInfo == true, | ^MappedPropertyInfo(type = MappedPropertyType.Enumeration, multiplicity = $propertyInfo.multiplicity), | [])),
+            name = $propertyInfo.name, enumPath = $type->elementToPath(), info = if($config.returnMappedPropertyInfo == true,
+            | ^MappedPropertyInfo(
```
Comment (gs-jp1) [inline legend-engine-xts-analytics/legend-engine-xts-analytics-mapping/legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverage/mappedEntityBuilder.pure:95]: minor: can we sort out indenting

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**